src/popover/popover.jsx

```diff
@@ -387,13 +383,20 @@ const Popover = React.createClass({
 
   render() {
     return (
-      <RenderToLayer
-        ref="layer"
-        open={this.state.open}
-        componentClickAway={this.componentClickAway}
-        useLayerForClickAway={this.props.useLayerForClickAway}
-        render={this.renderLayer}
-      />
+      <noscript>
```

Comment:
Let me know what you think about this, the children component both return null, I wasn't sure what the right tag was to use for something that ultimately wasn't rendering anything. I settled for noscript (which is what I think React uses when you return null) but if you'd prefer div or something else let me know.